integration_test.go

```diff
@@ -52,10 +52,12 @@ func testCartographerMap(t *testing.T, svc slam.Service, localizationMode bool)
 }
 
 func testCartographerPosition(t *testing.T, svc slam.Service, expectedComponentRef string) {
-	expectedPosOSX := r3.Vector{X: -4.534156132500952, Y: 9.800089705182817, Z: 0}
-	expectedPosLinux := r3.Vector{X: -3.0874594959315647, Y: 4.502970388777862, Z: 0}
+	expectedPosOSX := r3.Vector{X: 155.7488316264227, Y: -90.25868252233964, Z: 0}
+	expectedPosLinux := r3.Vector{X: 158.79903385710674, Y: -77.01514065531592, Z: 0}
 	tolerancePos := 0.001
-	expectedOri := &spatialmath.R4AA{Theta: 0.0018812019022632894, RX: 0, RY: 0, RZ: -1}
+
+	expectedOriOSX := &spatialmath.R4AA{Theta: 1.5465081272043815, RX: 0, RY: 0, RZ: 1}
+	expectedOriLinux := &spatialmath.R4AA{Theta: 0.3331667853231311, RX: 0, RY: 0, RZ: 1}
```

Comment:
After updating the dataset the orientations also differed slightly between architectures. The theta value's units are degrees as per the [proto](https://github.com/viamrobotics/api/blob/69f8be1dc959587244c71e044d54b004be2d9c77/common/v1/common.pb.go#L324C10-L324C10). This seems to be on a similar order of magnitude of error as a few millimeters like in the position, but want to make sure that orientation varying slightly does not mean we need to investigate any further

Reply:
I think its unsurprising that the orientation differs slightly on mac & linux in the because the position differs between platforms.

This can stay as is.